### PR TITLE
Nsfw guilds

### PR DESCRIPTION
**Pull request by IDoEverything**
# Summary
Adds the `nsfw` property to guilds

### CODE REVIEW

**Review by Naamloos — APPROVED** [review]
LGTM

File: DSharpPlus/Entities/Guild/DiscordGuild.cs [hunk 1]
```diff
@@ -396,6 +396,12 @@ public string BannerUrl
         [JsonProperty("premium_subscription_count", NullValueHandling = NullValueHandling.Ignore)]
         public int? PremiumSubscriptionCount { get; internal set; }
 
+        /// <summary>
+        /// Gets whether this guild is designated as nsfw.
+        /// </summary>
+        [JsonProperty("nsfw", NullValueHandling = NullValueHandling.Ignore)]
+        public bool Nsfw { get; internal set; }
```
Comment (uwx) [inline DSharpPlus/Entities/Guild/DiscordGuild.cs:403]: Property name should be `IsNsfw` to match other boolean properties

**Review by uwx — CHANGES_REQUESTED** [review]

File: DSharpPlus/Entities/Guild/DiscordGuild.cs (hunk 1 above)
Comment (uwx) [reply DSharpPlus/Entities/Guild/DiscordGuild.cs:403]: Actually, `IsNSFW`, that's what it is for channels.

**Review by uwx — APPROVED** [review]

File: DSharpPlus/Entities/Guild/DiscordGuild.cs [hunk 2]
```diff
@@ -396,6 +396,12 @@ public string BannerUrl
         [JsonProperty("premium_subscription_count", NullValueHandling = NullValueHandling.Ignore)]
         public int? PremiumSubscriptionCount { get; internal set; }
 
+        /// <summary>
+        /// Gets whether this guild is designated as nsfw.
```
Comment (Neuheit) [inline DSharpPlus/Entities/Guild/DiscordGuild.cs:400]:
```suggestion
        /// Gets whether this guild is designated as NSFW.
```

**Review by Neuheit — APPROVED** [review]

**Pull request merged**